packages/base-docs/src/DocPlugin.ts

```diff
@@ -50,7 +50,6 @@ export class DocPlugin extends Plugin<DocPluginObserve> {
     initialize(): void {
         this._localeService.getLocale().load({
```

Comment:
Is it still necessary to load i18n files in plugins? Since we have already load i18n files in the top-level Univer.